src/Extractor/HiveResultWriter.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\Extractor;
+
+use InvalidArgumentException;
+use Keboola\Csv\CsvWriter;
+use Keboola\DbExtractor\Adapter\ODBC\OdbcQueryResult;
+use Keboola\DbExtractor\Adapter\ResultWriter\DefaultResultWriter;
+use Keboola\DbExtractor\Adapter\ValueObject\ExportResult;
+use Keboola\DbExtractor\Adapter\ValueObject\QueryResult;
+use Keboola\DbExtractorConfig\Configuration\ValueObject\ExportConfig;
+
+class HiveResultWriter extends DefaultResultWriter
```

Comment:
Bolo potrebne implementovat `HiveResultWriter`, kedze Hive ma aj binarny typ, a ten potrebujeme previest do `base64`, aby nam to nerozbilo CSV.